cirrus/server/requirements.txt

```diff
@@ -1,2 +1,7 @@
 fastapi==0.92.0
-uvicorn==0.20.0
\ No newline at end of file
+uvicorn==0.20.0
+
+#tests
+pytest==7.2.2
+httpx==0.23.3
+pytest-cov==4.0.0
```

Comment:
Is there a reason you are not using hashes here?

Reply:
To be honest I don't have a strong reason why I haven't used hashes, that's only my personal choice I guess , but would love to hear your thoughts :)